src/bio/bio_xstream.c

```diff
@@ -108,6 +169,9 @@ bio_spdk_env_init(void)
 
 	opts.env_context = (char *)dpdk_cli_override_opts;
 
+	bio_get_hotplug_busid_range(nvme_glb.bd_nvme_conf);
+	spdk_nvme_pcie_set_hotplug_filter(hotplug_filter_fn);
```

Comment:
Minor: All these hotplug filter related code could be moved into bio_config.c, and export only single function like 'bio_set_hotplug_filter(nvme_conf)' in bio_internal.h.

Reply:
this is a lot tidier thanks for the suggestion